lib/galaxy/managers/libraries.py

```diff
@@ -145,7 +145,7 @@ def list(self, trans, deleted=False):
             allowed_library_manage_ids = set([])
             for action in all_actions:
                 if action.action == library_access_action:
-                    accessible_restricted_library_ids.add(action.library_id)
+                    accessible_restricted_library_ids.append(action.library_id)
```

Comment:
Or change also `accessible_restricted_library_ids` to a set?

Reply:
I vaguely remember there was a reason for this at one point. I don't think that is a bug though, so if you feel inclined please open a PR against dev.

Reply:
Not a bug, but `in` (and I suppose also sqlalchemy's `in_`) should be a lot faster for big sets then big lists.

Reply:
The actual code comes from https://github.com/galaxyproject/galaxy/pull/5542

Reply:
I am all for speed, but at this time of imminent release I do not want to wait for another round of tests. I opened a PR with this suggestion against dev in https://github.com/galaxyproject/galaxy/pull/5681.